velox/docs/functions/spark/array.rst

```diff
@@ -86,6 +86,16 @@ Array Functions
         SELECT array_sort(array(NULL, 1, NULL)); -- [1, NULL, NULL]
         SELECT array_sort(array(NULL, 2, 1)); -- [1, 2, NULL]
 
+.. spark:function:: array_shuffle(array(E), seed) -> array(E)
+
+    Generates a random permutation of the given ``array`` using a seed derived 
+    from the user-specified ``seed`` and the configuration `spark.partition_id`.
+    ``seed`` must be constant. ::
+
+        SELECT array_shuffle(ARRAY [1, 2, 3], 0); -- [3, 1, 2]
```

Comment:
This is not a valid SQL in Spark. 
```
spark-sql (default)> SELECT shuffle(ARRAY (1, 2, 3), 0);
[WRONG_NUM_ARGS.WITHOUT_SUGGESTION] The `shuffle` requires 1 parameters but the actual number is 2. Please, refer to 'https://spark.apache.org/docs/latest/sql-ref-functions.html' for a fix.; line 1 pos 7
spark-sql (default)> SELECT shuffle(ARRAY (1, 2, 3));
[3,2,1]
Time taken: 0.193 seconds, Fetched 1 row(s)
spark-sql (default)> SELECT shuffle(ARRAY (1, 2, 3,5));
[3,1,2,5]
Time taken: 0.139 seconds, Fetched 1 row(s)
```
We could change to
`
SELECT array_shuffle(array(1, 2, 3), 0); -- [3, 1, 2]`

Reply:
Yes, this is not a valid SQL in spark, spark generated the seed and pass to this function, use can't set the seed.

Reply:
ARRAY [1, 2, 3]  -> array(1, 2, 3)